skymp5-front/src/constructorComponents/chat/index.js

```diff
@@ -135,46 +139,59 @@ const Chat = (props) => {
     ));
   };
 
-
   return (
-    <div id="chat">
-      <div className="chat-main">
-        <div className={`list ${hideNonRP ? 'hideNonRP' : ''}`} ref={chatRef} onScroll={(e) => handleScroll()}>{getList()}</div>
-        {isInputHidden
-          ? <></>
-          : <div className='input'>
-            <div>
-              <input
-                id="chatInput"
-                className={'show'}
-                type="text"
-                placeholder={placeholder !== undefined ? placeholder : ''}
-                value={input}
-                onChange={(e) => { updateInput(e.target.value); }}
-                onFocus={(e) => changeInputFocus(true)}
-                onBlur={(e) => changeInputFocus(false)}
-                ref={inputRef}
-              />
-            </div>
-            <div className='chat-checkboxes'>
-              <ChatCheckbox id={'nonrp'} text={'non-rp'} isChecked={hideNonRP} onChange={(e) => changeNonRPHide(e.target.checked)} />
-              <ChatCheckbox id={'diceSound'} text={'dice sounds'} isChecked={disableDiceSounds} onChange={(e) => setDisableDiceSounds(e.target.checked)} />
-              {/* Maybe we will need it later: <ChatCheckbox id={'diceColor'} text={'dice colors'} isChecked={!disableDiceColors} onChange={(e) => setDisableDiceColors(!e.target.checked)} /> */}
-              <span className={`chat-message-limit ${input.length > MAX_LENGTH ? 'limit' : ''} text`}>{input.length}/{MAX_LENGTH}</span>
+    <div className='fullPage'>
+      <Draggable handle='#handle' disabled={!moveChat} bounds={'.fullPage'}>
+        <div id="chat">
+          <div className="chat-main">
+            {
+              isInputHidden
+                ? null
+                : <div className='chat-corner'>
+                      <img src={ChatCorner} />
+                  </div>
+            }
```

Comment:
Afaik this can be replaced with
```ts
            {
              !isInputHidden &&
                  <div className='chat-corner'>
                      <img src={ChatCorner} />
                  </div>
            }
```